### DISCUSSION TITLE
Basic Time Series Analysis

### DISCUSSION THREAD

**Discussion by DFHydrogeologist** [pastas/pastas#758] [Q&A] [answered]
If I have a Groundwater time series but I don´t have any recharge or evaporation series for a stress model, is it possible to use pastas to analyze the data?

**Comment by raoulcollenteur** [comment, marked as answer]
Hi @DFHydrogeologist, Pastas assumes you want to model the heads using some stresses, often precipitation and evaporation, among others. These can be obtained for most places from open data sources. You can take a look at the paper describing the basic principles of Pastas here: https://ngwa.onlinelibrary.wiley.com/doi/full/10.1111/gwat.12925

**Comment by mbakker7** [comment]
You really need time series of the stresses (forcings, drivers) that cause the groundwater dynamics. But there are lots of databases around for many parts of the world. For Europe, for example, you can take a look at https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2017JD028200.
